openshift/EAP7-1378_Exclude_Galleon_layers.adoc

```diff
@@ -0,0 +1,108 @@
+= EAP7-1378 Exclude Galleon layers during s2i
+:author:            JF Denise
+:email:             jdenise@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          openshift,galleon,layers
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse/
+
+== Overview
+
+The intent of this feature is to allow to exclude optional galleon layers when provisioning
+galleon layers using _GALLEON_PROVISION_LAYERS=<comma separated list of layers>_ during s2i.
+
+NB: This feature is already present in latest image but unsupported and undocumented.
+WildFly s2i builder image already supports this feature (see community documentation). 
+
+== Issue Metadata
+
+=== Issue:
+
+* {issue-base-url}EAP7-1378[EAP7-1378]
+
+=== Related Issues:
+
+* {issue-base-url}CLOUD-3470[CLOUD-3470]
+
+* {issue-base-url}GAL-306[GAL-306]
+
+=== Dev Contacts:
+
+* mailto:jdenise@redhat.com[Jean-François Denise]
+
+=== QE Contacts:
+
+Testing done by Eng.
+
+* mailto:jdenise@redhat.com[Jean-François Denise]
+
+=== Affected Projects or Components:
+
+* Openshift EAP images
+
+
+== Requirements
+
+=== Hard requirements
+
+Generic mechanism:
+
+* Ability to exclude an optional layer from a provisioned layer.
+
+* Operated from the _GALLEON_PROVISION_LAYERS_ env variable during s2i. Any layer present in the env var value that
+starts with '-' are excluded from the set of provisioned layers.
+
+* Attempt to exclude a required layer leads to an error and s2i phase aborts.
+
+Although the mechanism to exclude layers is generic, we are only supporting a fixed set of layers one can exclude. This fixed set
+is a first step toward more flexibility (more layers to be excluded).
+
+NB: THIS COULD BE IMPACTED BY THE WAY WE EXPOSE MICROPROFILE LAYERS (Aggregation vs no aggregation).
+
+* datasources-web-server: _datasources_ layer can be excluded
+- _GALLEON_PROVISION_LAYERS=datasources-web-server,-datasources_
+
+* jaxrs-server: _jpa_ and _jpa,datasources_ can be excluded. NB: Excluding _datasources_ alone is not supported, _jpa_ requires it.
+- _GALLEON_PROVISION_LAYERS=jaxrs-server,-jpa_
+- _GALLEON_PROVISION_LAYERS=jaxrs-server,-jpa,-datasources_
+
+* cloud-server:  Supports the exclusions of _jaxrs-server_. In addition _observability_ and _open-tracing_ layers can be excluded
+- _GALLEON_PROVISION_LAYERS=_cloud-server,-observability,[jaxrs-server exclusions]_
```

Comment:
preview of the file is not correct. It is confusing to get `_` in the output (I assume that

Could you please use a code block to make sure there is no confusion on the syntax?